kafka-client/src/main/java/dev/responsive/kafka/internal/metrics/exporter/otel/OtelMetricsService.java

```diff
@@ -75,6 +75,7 @@ public static OtelMetricsService create(
       builder.addHeader(ApiKeyHeaders.SECRET_METADATA_KEY, secret.value());
     }
 
+    builder.setCompression("gzip");
```

Comment:
does anything need to happen on the reader side or is this handled by GRPC under the hood?

Reply:
+1 - were you able to test that the controller reads the metrics correctly?

Reply:
I ran it against the default open telemetry collector, and I assumed the compression was handled under the hood as usually happens with HTTP connections, but I'll test it with our own controller before merging.

Reply:
Just ran this stuff locally against our production controller and it works 👍